<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutInf</class>
 <widget class="QWidget" name="LayoutInf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Sitka Text</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>layout details</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>width:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="wEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>height:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>read only:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="readEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>surfaces:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="surfaceEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>windows:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="windowEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
